apps/andi/lib/andi/input_schemas/organizations.ex

```diff
@@ -91,9 +105,17 @@ defmodule Andi.InputSchemas.Organizations do
 
   def is_unique?(id, org_name) do
     from(org in Andi.InputSchemas.Organization,
-      where: org.orgName == ^org_name and org.id != ^id
+      where: org.orgName == ^org_name or org.id == ^id
```

Comment:
Am I insane, becuase it seems like this would be false every time?

Reply:
This works or making sure the id and org name are new when you create a new org. The only thing I just realized is it messes up the save when you are editing the org so I might have to put some thing into that